README.md

```diff
@@ -128,119 +89,25 @@ Work has begun on the MVP.
   * [ ] Deployment
 
 
-## Usage
-
-The software is far from being even an alpha quality. Most of the values and
-configuration is hard-coded inside the scripts. Once we prove this project,
-we'll start working on configuration and user experience. You've been warned.
+## Requirements
 
-There is a makefile in this repository which can be used to build the container
-images and run them locally with podman.
+Packit is written in python 3 and is supported only on 3.6 and later.
 
-Before that, you should create a file secrets.yaml and store your github and
-[pagure API tokens](https://src.fedoraproject.org/settings#nav-api-tab) in
-there. This file is then used during deployment as a source for ansible
-variables which are utilized by the containers:
-```yaml
-$ cat ./secrets.yaml
-github_token: 123456
-
-# This token is needed to access data in pagure.
-# This is meant to be an API token of your user
-# https://src.fedoraproject.org/settings#nav-api-tab
-pagure_user_token: 456789
-
-# We need this token to be able to comment on a pagure pull request.
-# If you don't have commit access to the package, you should ask maintainer of
-# the package to generate a token for the specific dist-git repository.
-# https://src.fedoraproject.org/rpms/<package>/settings#apikeys-tab
-pagure_package_token: ABCDEF
-
-# This token is needed to create a pull request
-# https://src.fedoraproject.org/fork/<user>/rpms/<package>/settings#apikeys-tab
-pagure_fork_token: QWERTY
-```
-
-The next thing you need, in order for the tooling to be able to push to your fork in Fedora dist-git, SSH keys.
-
-Please place two files to the root of this repo:
- * bigger-secret — public key
- * biggest-secret — private key
-
-Both of them are in .gitignore so they will not land in git.
-
-
-Let's build and run now:
-```
-$ make build
-$ make run-local
-```
 
-## Local CLI installation
+## Installation
 
 You can install packit using `pip`:
 
 ```
 $ pip3 install --user git+https://github.com/packit-service/packit.git
 ```
 
-Please bear in mind that packit works only with python 3.6.
-
-```
-$ packit --help
-Usage: packit [OPTIONS] COMMAND [ARGS]...
-
-Options:
-  -d, --debug
-  --fas-user TEXT
-  -k, --keytab TEXT
-  -v, --verbose
-  -h, --help         Show this message and exit.
-
-Commands:
-  propose-update   Release current upstream release into Fedora
-  sg2dg            Convert source-git repo to dist-git repo.
-  srpm             Generate a srpm from packit.
-  version          Display the version.
-  watch-fedora-ci  watch for flags on PRs: try to process those...
-  watch-pr         watch for activity on github and...
+Packit depends on multiple libraries and tools. We suggest to install
+`rebase-helper` from Fedora repositories, since the tool is not on PyPI:
```

Comment:
Rebase-helper was released to PyPI yesterday..;-)

Reply:
thanks, updated